03/app.js

```diff
@@ -1,5 +1,48 @@
 document.addEventListener('DOMContentLoaded', init);
 
 function init() {
-    console.log('DOM');
+    updateIP();
+};
+
+
+// po kliknięciu w BUTTON drugi raz otrzymuję error.
+// ** Failed to execute 'text' on 'Response': body stream is locked
+// at app.js: 22
+// CZY to dlatego ze promise mozna skonsumowac tylko raz? jesli tak, to jak to naprawic ze kazde klikniecie bedzie mi akutualizowalo ID?
+
+// dodaj funkcjonalnosc wielokrotnego klikania na button
+
+
+function updateIP() {
+    const btn = _getHTMLElement('BUTTON');
+    const apiIP = fetch('https://api.wheretheiss.at/v1/satellites/25544');
+
+    btn.addEventListener('click', () => handleRequest(apiIP));
+};
+
+
+async function handleRequest(url) {
```

Comment:
skoro async to powiniśmy zwracać jakaś wartość i pewnie użyć gdzieś `await`. Tutaj raczej jest to zbędne :)

Reply:
OK